debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -353,6 +353,10 @@ protected static void dropPublication() {
         dropPublication(ReplicationConnection.Builder.DEFAULT_PUBLICATION_NAME);
     }
 
+    protected static void createPublicationForAllTables() {
+        dropPublication(ReplicationConnection.Builder.DEFAULT_PUBLICATION_NAME);
```

Comment:
Is it correct, that method `createPublicationForAllTables()` actually does not create any publications and just drops default publication?